CHANGELOG.txt

```diff
@@ -4,6 +4,7 @@ Changelog
 4.0 (xx.xx.xxxx) - IN DEVELOPMENT
 ~~~~~~~~~~~~~~~~
 
+* Added a new ``BaseGenericSetting`` base model class that allows defining a settings model that applies to all sites rather than just a single site (Kyle Bayliss)
```

Comment:
Please use single backticks here...
```suggestion
* Added a new `BaseGenericSetting` base model class that allows defining a settings model that applies to all sites rather than just a single site (Kyle Bayliss)
```